src/metabase/formatter/datetime.clj

```diff
@@ -61,9 +61,9 @@
                               ;; and not have any metadata. Since we don't know the metadata, we can never
                               ;; match a key with metadata, even if we do have the correct name or id
                               (update-keys #(select-keys % [::mb.viz/field-id ::mb.viz/column-name])))]
-    (or (all-cols-settings {::mb.viz/field-id field-id-or-name})
-        (all-cols-settings {::mb.viz/column-name field-id-or-name})
-        (all-cols-settings {::mb.viz/column-name column-name}))))
+    (merge (all-cols-settings {::mb.viz/field-id field-id-or-name})
+           (all-cols-settings {::mb.viz/column-name field-id-or-name})
+           (all-cols-settings {::mb.viz/column-name column-name}))))
```

Comment:
Should these latter two `::mb.viz/column-name` lookups be a nested `or`, like we discussed elsewhere?